twisted/trial/util.py

```diff
@@ -362,7 +362,7 @@ def _unusedTestDirectory(base):
             # Create it anew and mark it as ours so the next _removeSafely on
             # it succeeds.
             testdir.makedirs()
-            testdir.child('_trial_marker').setContent(b'')
```

Comment:
https://github.com/twisted/twisted/compare/trunk...moar-windows-8025-7#diff-c7ada4588c4ea33358407df5c6f0d592L1535 is where I have partially fixed it; I need to push that branch forward.